packages/platform-sdk-ada/src/services/client.ts

```diff
@@ -41,7 +43,7 @@ export class ClientService extends Services.AbstractClientService {
 		);
 
 		return Helpers.createTransactionDataCollectionWithType(
-			transactions,
+			transactions.map(transaction => [transaction, this.#decimals]),
```

Comment:
This is awkward

Reply:
See https://github.com/ArkEcosystem/platform-sdk/pull/1666#discussion_r643977704